expression/builtin_encryption_vec.go

```diff
@@ -330,11 +331,59 @@ func (b *builtinSHA1Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column)
 }
 
 func (b *builtinUncompressSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalString evals UNCOMPRESS(compressed_string).
+// See https://dev.mysql.com/doc/refman/5.7/en/encryption-functions.html#function_uncompress
 func (b *builtinUncompressSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	result.ReserveString(n)
+	sc := b.ctx.GetSessionVars().StmtCtx
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+
+		payload := buf.GetString(i)
+
+		if len(payload) == 0 {
+			result.AppendNull()
```

Comment:
Maybe here we should do ```result.AppendString("")``` ? What's your opinion? @qw4990  @shihongzhi

Reply:
Yes, you are right.
Please fix this problem @shihongzhi .